internal/locker/locker.go

```diff
@@ -94,7 +91,7 @@ func (d *debugger) Lock() {
 	d.holder = string(buf)
 
 	d.timer = time.AfterFunc(5*time.Second, func() {
-		d.logger.Printf("Potential dead lock detected for a lock %q held by: %v\n", d.name, d.holder)
+		logger.Debugf("debug_mutex: Potential dead lock detected for a lock %q held by: %v\n", d.name, d.holder)
```

Comment:
I am approving these PRs for now. But we need a better way to append prefixes. Please create a bug and track this.